libfwupdplugin/fu-dpaux-device.c

```diff
@@ -129,13 +129,10 @@ fu_dpaux_device_setup(FuDevice *device, GError **error)
 	priv->dpcd_dev_id = fu_struct_dpaux_dpcd_get_dev_id(st);
 	fu_device_set_version_raw(device, fu_struct_dpaux_dpcd_get_fw_ver(st));
 
-	/* sanity check */
+	/* sanity check, for broader display support it's not enforced */
 	if (priv->dpcd_ieee_oui == 0x0) {
-		g_set_error_literal(error,
-				    FWUPD_ERROR,
-				    FWUPD_ERROR_NOT_SUPPORTED,
-				    "no DPCD OUI data was provided");
-		return FALSE;
+		g_debug("DPAUX device has no IEEE OUI implemented");
+		return TRUE;
```

Comment:
Didn't we just start doing this because algotek was missing one?

Are you saying we have a mismatch of expectations for multiple vendors?

Reply:
Yes, another mismatch from mediatek_scaler device. This probably not a bottom bar for display controller.

Reply:
@hughsie, thoughts?

Reply:
I'm not against it -- I just don't want a DPAUX/OUI_00000000 instance ID. Maybe rather than returning TRUE here we can just remove the if block and do this instead:

	-       fu_device_add_instance_u32(device, "OUI", priv->dpcd_ieee_oui);
	+       if (priv->dpcd_ieee_oui != 0)
	+               fu_device_add_instance_u32(device, "OUI", priv->dpcd_ieee_oui);

which means that all the fu_device_build_instance_id_full() silently fail and we fall through to the end.

Reply:
Updated, thanks.